pkg/utils/predicates.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
In fact, you can add these functions to node_affinity.go and remove this file.

Reply:
Sure, but what I was thinking was whenever the the scheduler becomes importable, then all we need to do is import it and remove this file. I didn't want to change a lot of code from these functions for the same reason, WDYT?